node/src/test/test-ActiveSpeakerObserver.ts

```diff
@@ -27,8 +27,14 @@ beforeEach(async () => {
 	ctx.router = await ctx.worker.createRouter({ mediaCodecs: ctx.mediaCodecs });
 });
 
-afterEach(() => {
+afterEach(async () => {
 	ctx.worker?.close();
+
+	if (ctx.worker?.subprocessClosed === false) {
```

Comment:
We check ' === false` because if the worker fails to be created, then `ctx.worker?.subprocessClosed` will return `undefined`. In that case we do NOT want to await for any event.